packages/core/upload/admin/src/pages/App/tests/MediaLibrary.test.js

```diff
@@ -350,8 +350,7 @@ describe('Media library homepage', () => {
 
       renderML();
 
-      expect(screen.queryByText('list.folders.title')).not.toBeInTheDocument();
-      expect(screen.queryByText('Folder 1')).not.toBeInTheDocument();
+      expect(useFolders).toHaveBeenCalledWith({ enabled: false, query: { _q: 'true' } });
```

Comment:
To make it clearer that we are actually only interested in the enabled prop, what do you think about using?

```suggestion
      expect(useFolders).toHaveBeenCalledWith(expect.objectContaining({ enabled: false }));
```

The test doesn't really care what is being passed in for the others and it can be a source of frustration if it needs to be changed over and over again.

Reply:
didn't know about this one `objectContaining`, makes sense!`